prov/efa/src/rdm/efa_rdm_protocol.h

```diff
@@ -233,7 +233,7 @@ struct efa_rdm_readrsp_hdr {
 	uint64_t seg_length;
 };
 
-EFA_RDM_ENSURE_HEADER_SIZE(efa_rdm_ctsdata_hdr, sizeof (struct efa_rdm_ctsdata_hdr));
+EFA_RDM_ENSURE_HEADER_SIZE(efa_rdm_readrsp_hdr, sizeof(struct efa_rdm_ctsdata_hdr));
```

Comment:
I don't quite follow... why ensure efa_rdm_readrsp_hdr has a size of efa_rdm_ctsdata_hdr?

Reply:
This was the oldest commit I could find: https://github.com/ofiwg/libfabric/blame/ab4ef648c1288b0459e4ced9c5363db9dd48b6dd/prov/efa/src/rdm/efa_rdm_protocol.h#L268-L270

No comment on why it needs to be the same size

Reply:
@wzamazon  is this by design?

Reply:
I think it was because the same function was used to handle READRSP and CTSDATA

Reply:
Now that we have different function to handle them I do not this this assertion is necessary any more.

Reply:
Changed it back to 24 bytes